infrastructure/kctf/samples/apache-php/challenge/config/kustomization.yaml

```diff
@@ -0,0 +1,12 @@
+configMapGenerator:
+- name: challenge-skeleton-config
+  files:
+  - pow
```

Comment:
pow is not actually supported, and for web tasks it might not actually make sense anyway

Reply:
any reason not to remove it in that case?